pyproject.toml

```diff
@@ -28,6 +28,12 @@ stringcase = "^1.2.0"
 docker = "^7.1.0"
 sqlalchemy = "~=1.0"
 requests = "^2"
+wrapt = "^1.16.0"
+opentelemetry-sdk = "^1.25.0"
+opentelemetry-api = "^1.25.0"
+opentelemetry-instrumentation = "^0.46b0"
+opentelemetry-instrumentation-threading = "^0.46b0"
+opentelemetry-exporter-otlp = "^1.25.0"
```

Comment:
I think we need to figure out a way to get these as optional dependencies. Feels like a lot of baggage out of box.

Reply:
Yeah, I'm planning on adding an extra, will happen in a subsequent draft (this is nice reminder!)

Reply:
Done